### PR TITLE
added mess menu generating script

### PR DESCRIPTION
**Pull request by MistyRavager**
creating the folder bus-menu-script and added mess menu script into it.

### CODE REVIEW

File: bus-menu-scripts/mess.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+days = ["Sunday", "Monday", "Tuesday",
+        "Wednesday", "Thursday", "Friday", "Saturday"]
+meals = ["Breakfast", "Lunch", "Snacks", "Dinner"]
+
+regular_items = {}
+extra_items = {}
+daily_items = {}
+
+for day in days:
```
Comment (RachitKeertiDas) [inline bus-menu-scripts/mess.py]: You can do this in a slightly better fashion using `dict.fromkeys()`

Something like `regular_items = dict.fromkeys(days,[])` might work

File: bus-menu-scripts/mess.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    regular_items[day] = {}
+    extra_items[day] = {}
+for meal in meals:
```
Comment (RachitKeertiDas) [inline bus-menu-scripts/mess.py]: Similar to the above comment.

File: bus-menu-scripts/mess.py (hunk 1 above)
Comment (MistyRavager) [reply bus-menu-scripts/mess.py]: Previously I had put `reg_items = dict.fromkeys(days,[])`, but the problem is that this creates a shallow copy of `[]` resulting in addition of an item in every value of the dictionary

Comment (RachitKeertiDas) [reply bus-menu-scripts/mess.py]: Oh yeah, nice catch.

You can use dict comprehension instead. Something like `regular_items = {day:[] for day in days}`

Comment (MistyRavager) [reply bus-menu-scripts/mess.py]: Added the dict comprehensions.

**Pull request merged**